src/app/app.component.ts

```diff
@@ -61,10 +62,16 @@ export class AppComponent implements OnInit {
                     this.gateway['config']['api_url'] = apiHost;
                 }
 
+                if(this.gateway['config']['ver']){
+                    this.version = this.gateway['config']['ver'];
+                }else{
+                    this.version = 'v1';
+                }
+
```

Comment:
Please following the coding style of existing code base.

Reply:
These changes are in-order to serve old version request.

Reply:
fixed